mitosheet/src/components/checklists/Checklist.tsx

```diff
@@ -0,0 +1,268 @@
+    // As of now, we only have a single checklist, so we just check to see how much of this we have done
+    const completedItems = userProfile.receivedChecklists['onboarding_checklist'] || [];
+    return CHECKLIST_STEPS['onboarding_checklist'].filter(checklistItem => !completedItems.includes(checklistItem))
+}
+
+const getChecklistIcon = (remainingChecklistItems: string[]): JSX.Element => {
+    if (remainingChecklistItems.length === 1) {
+        return (<Checklist5Icon/>)
+    } else if (remainingChecklistItems.length === 2) {
+        return (<Checklist4Icon/>)
+    } else if (remainingChecklistItems.length === 3) {
+        return (<Checklist3Icon/>)
+    } else if (remainingChecklistItems.length === 4) {
+        return (<Checklist2Icon/>)
+    } 
+    return (<Checklist1Icon/>)
+}
+
+const getChecklistItemTitle = (item: string): string => {
+    if (item === 'signup') {
+        return 'Sign up for Mito'
+    } else if (item === 'import') {
+        return 'Import data'
+    } else if (item === 'filter') {
+        return 'Filter your dataframe'
+    } else if (item === 'pivot') {
+        return 'Generate a pivot table'
+    } else if (item === 'graph') {
+        return 'Create a graph'
+    }
+    return 'Continue exploring'
+}
+
+const getChecklistItemLink = (item: string): string => {
+    if (item === 'signup') {
+        return 'https://docs.trymito.io/how-to/creating-a-mitosheet'
+    } else if (item === 'import') {
+        return 'https://docs.trymito.io/how-to/importing-data-to-mito'
+    } else if (item === 'filter') {
+        return 'https://docs.trymito.io/how-to/filter-data'
+    } else if (item === 'pivot') {
+        return 'https://docs.trymito.io/how-to/pivot-tables'
+    } else if (item === 'graph') {
+        return 'https://docs.trymito.io/how-to/graphing'
+    }
+    return 'https://docs.trymito.io/how-to/search-for-functionality'
+}
+
+
+const ChecklistItem = (props: {
+    index: number,
+    text: string,
+    icon: JSX.Element,
+    href: string
+}): JSX.Element => {
+    return (
+        <Row 
+            justify='space-between' 
+            align='center' 
+            className='text-body-1 text-color-white-important text-underline-on-hover mt-5px'
+            onClick={() => {
+                window.open(props.href, '_blank');
+            }}
+        > 
+            <Col>
+                {props.index + 1}. {props.text}
+            </Col>
+            <Col>
+                {props.icon}
+            </Col>
+        </Row>
+    )
+}
+
+
+
+const ChecklistTODOItem = (props: {
+    index: number,
+    item: string,
+    completed: boolean,
+}): JSX.Element => {
+    return (
+        <ChecklistItem
+            index={props.index}
+            text={getChecklistItemTitle(props.item)}
+            icon={props.completed ? <CheckmarkIcon/> : <EmptyCircleIcon/>}
+            href={getChecklistItemLink(props.item)}
+        />
+    )
+}
```

Comment:
It seems like the ChecklistTODOItem isn't that helpful of a component? Want to just remove it and use the ChecklistItem directly?

Reply:
Ahh I guess it is being used for the TODO item and next step item